### DISCUSSION TITLE
[Support]: Upgrading ... to last release [stable]

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#13299] [General Support] [answered]
### Describe the problem you are having

Hi, 
I would upgrade  frigate (on proxmox LXC), 
I just done a copy using SCP[1] for frigate.db file
of course a copy of config.yml.
Now, I think to edit the docker-compose.yaml 
these following lines, that are different from the .13 version
```
ports:
      - "8971:8971"
      # - "5000:5000" # Internal unauthenticated access. Expose carefully.
      - "8554:8554" # RTSP feeds
      - "8555:8555/tcp" # WebRTC over tcp
      - "8555:8555/udp" # WebRTC over udp
```
and so I could recreate the docker or... any idea to prevent a disaster ? :)
Actually frigate works flawless but I too curious to give a look to the new UI and last features.
(docker compose and Portainer are installed).

[1]: scp frigate.db   admin@yourIPfolder:/frigate.db 
(so no mounting point needed and it works)
Thank you!

### Version

0.13.2-6476f8a

### Frigate config file

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Operating system

Proxmox

### Install method

Docker CLI

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

n/a

**Comment by NickM-27** [comment, marked as answer]
as long as you have a backup, then all that is needed is to pull the new image and apply it via docker

**Comment by phlnil** [comment]
Thank you NickM-27!
_for supernewbie like me:_ 
```
in the right folder :
# docker compose pull
# docker compose up -d
```
correctly update to the last stable version !
